src/logic/hooks/useEstimateTransactionGas.tsx

```diff
@@ -31,6 +31,9 @@ export enum EstimationStatus {
 const checkIfTxIsExecution = (threshold: number, preApprovingOwner?: string, txConfirmations?: number): boolean =>
   txConfirmations === threshold || !!preApprovingOwner || threshold === 1
 
+const checkIfTxIsApproveAndExecution = (threshold: number, txConfirmations: number): boolean =>
+  txConfirmations + 1 === threshold
```

Comment:
not sure to understand this, are we checking the checkbox status somewhere?

Reply:
It's not needed, it's the same logic that uses the checkbox in `ApproveTxModal` here:
```
  const oneConfirmationLeft = !thresholdReached && tx.confirmations.size + 1 === threshold
  const isTheTxReadyToBeExecuted = oneConfirmationLeft ? true : thresholdReached
```